modules/apps/object/object-admin-rest-impl/rest-openapi.yaml

```diff
@@ -732,6 +732,71 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/ObjectDefinition"
             tags: ["ObjectDefinition"]
+    "/object-definitions/by-external-reference-code/{objectDefinitionExternalReferenceCode}/object-layouts":
```

Comment:
Hey @gabrielwas,
Could you please explain why we should name "objectDefinitionExternalReferenceCode" if the path already indicates that is from the object definition ("/object-definitions/by-external-reference-code/...") ?

Reply:
There is already a path "/object-definitions/by-external-reference-code/{externalReferenceCode}" in the object definition.

Reply:
Hey @marcelabc !

Good catch! I've checked some patterns related to it on portal (search for `/by-external-reference-code/`) and I could see that we put the "full name" when we have multiple externalReferenceCodes in the same path.

Ex:` /accounts/by-external-reference-code/{accountExternalReferenceCode}/account-roles/{accountRoleId}/user-accounts/by-external-reference-code/{userAccountExternalReferenceCode}`

So, in this case, it should be externalReferenceCode.

@p-albuquerque, could you change it, please. Thx